packages/itwinui-react/src/core/SearchBox/SearchBox.test.tsx

```diff
@@ -86,7 +86,7 @@ it('should render expandable Searchbox', async () => {
   expect(searchbox).not.toHaveAttribute('data-iui-expanded', 'true');
 
   const openButton = searchbox?.querySelector(
-    '.iui-searchbox-open-button',
+    '.iui-button.iui-input-flex-container-icon',
```

Comment:
nit: just `'button'` should be enough as a selector. No reason to tie this test to the CSS classes (which might change again)

Reply:
Makes sense, done (https://github.com/iTwin/iTwinUI/pull/1888/commits/baf0fc874f93ef0f2a89c7f0bc752d5ce3ce581c).

Reply:
No I mean just `querySelector("button")`. Right now you are still relying on `iui-input-flex-container-icon`, which isn't even part of the `SearchBox`.

And if you have multiple buttons and need further disambiguation, using a custom class/attribute should be preferred.

With all tests written in this generic way, changing a CSS class would result in fewer unit test changes.

Reply:
Ohh, I see. Changed (https://github.com/iTwin/iTwinUI/pull/1888/commits/a018e0b21741fa3a492c7841e97fdc2ecd3bd6d4).